Dockerfile

```diff
@@ -90,31 +90,12 @@ ENTRYPOINT [ "/usr/bin/tini", "--" ]
 
 CMD [ "/opt/netbox/docker-entrypoint.sh", "/opt/netbox/launch-netbox.sh" ]
 
-LABEL netbox.original-tag="" \
-      netbox.git-branch="" \
-      netbox.git-ref="" \
-      netbox.git-url="" \
-# See http://label-schema.org/rc1/#build-time-labels
-# Also https://microbadger.com/labels
-      org.label-schema.schema-version="1.0" \
-      org.label-schema.build-date="" \
-      org.label-schema.name="NetBox Docker" \
-      org.label-schema.description="A container based distribution of NetBox, the free and open IPAM and DCIM solution." \
-      org.label-schema.vendor="The netbox-docker contributors." \
-      org.label-schema.url="https://github.com/netbox-community/netbox-docker" \
-      org.label-schema.usage="https://github.com/netbox-community/netbox-docker/wiki" \
-      org.label-schema.vcs-url="https://github.com/netbox-community/netbox-docker.git" \
-      org.label-schema.vcs-ref="" \
-      org.label-schema.version="snapshot" \
 # See https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys
-      org.opencontainers.image.created="" \
-      org.opencontainers.image.title="NetBox Docker" \
+LABEL org.opencontainers.image.title="NetBox Docker" \
       org.opencontainers.image.description="A container based distribution of NetBox, the free and open IPAM and DCIM solution." \
       org.opencontainers.image.licenses="Apache-2.0" \
       org.opencontainers.image.authors="The netbox-docker contributors." \
       org.opencontainers.image.vendor="The netbox-docker contributors." \
       org.opencontainers.image.url="https://github.com/netbox-community/netbox-docker" \
       org.opencontainers.image.documentation="https://github.com/netbox-community/netbox-docker/wiki" \
-      org.opencontainers.image.source="https://github.com/netbox-community/netbox-docker.git" \
-      org.opencontainers.image.revision="" \
-      org.opencontainers.image.version="snapshot"
```

Comment:
The thing is that if someone does not build it with the build script, then the labels are missing. That's why they were declared here as well. And that's why I would leave them here, even if they are defined in the buildscript.

Also, it's easier to identify all the labels that should be available if they are all in one place, compared to when you have to look into the build script and into the Dockerfile.

Reply:
Ok, I will leave them there.